bitmind/image_transforms.py

```diff
@@ -80,18 +82,20 @@ def __call__(self, img):
         return img
 
 
-# transforms to prepare an image for the base miner.
+# transforms to prepare an image for the base miner
 base_transforms = transforms.Compose([
     CenterCrop(),
-    transforms.Resize((224, 224)),
+    transforms.Resize(TARGET_IMAGE_SIZE),
     transforms.ToTensor(),
     transforms.Lambda(lambda t: t.expand(3, -1, -1) if t.shape[0] == 1 else t),
 ])
 
-# example data augmentation
-random_image_transforms = ComposeWithParams([
-    RandomResizedCropWithParams(256, scale=(0.2, 1.0), ratio=(1.0, 1.0)),
+# data augmentation
+random_aug_transforms = ComposeWithParams([
+    transforms.ToTensor(),
+    transforms.Lambda(lambda t: t.expand(3, -1, -1) if t.shape[0] == 1 else t),
```

Comment:
used to concatenate these onto the base transforms, now they're kept independent

Reply:
Looks good, had the same question as Andrew about either dropping 4 channel images (dropping the alpha channel) or some other approach.